docs/02-quick-start/01-installation/02-kubernetes.md

```diff
@@ -1,7 +1,52 @@
 ---
 sidebar_label: Kubernetes
 ---
+<!---
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
 
 # Try Ozone With Kubernetes
 
-**TODO:** File a subtask under [HDDS-9856](https://issues.apache.org/jira/browse/HDDS-9856) and complete this page or section.
+**Requirements:**
+
+* Working kubernetes cluster (LoadBalancer, PersistentVolume are not required)
+* kubectl
+
+As the *apache/ozone* Docker images are available from the dockerhub the deployment process is very similar to Minikube deployment. The only big difference is that we have dedicated set of k8s files for hosted clusters (for example we can use one Datanode per host)
+Deploy to kubernetes
```

Comment:
This is supposed to be a heading (was wrong in the existing docs).

```suggestion
### Deploy to Kubernetes
```

(Suggesting 3rd level heading, assuming Minikube and "hosted cluster" content would be 2nd level, and this is under "hosted cluster".)